core/rpc-server/src/lib.rs

```diff
@@ -59,8 +59,20 @@ where
 			.rpc(enclave_req.as_bytes().to_vec())
 			.map_err(|e| CallError::Failed(e.into()))
 	})?;
-
-	server.register_module(module).unwrap();
+	server.register_module(import_sidechain_block_module).unwrap();
+
+	let mut fetch_sidechain_blocks_module = RpcModule::new(sidechain_block_fetcher);
+	fetch_sidechain_blocks_module.register_method(
+		"sidechain_fetchBlocksFromPeer",
+		|params, sidechain_block_fetcher| {
+			debug!("sidechain_fetchBlocksFromPeer: {:?}", params);
+			let (block_hash, shard_identifier) = params.one::<(BlockHash, ShardIdentifier)>()?;
+			sidechain_block_fetcher
+				.fetch_all_blocks_following(&block_hash, &shard_identifier)
+				.map_err(|e| CallError::Failed(e.into()))
+		},
+	)?;
+	server.register_module(fetch_sidechain_blocks_module).unwrap();
```

Comment:
The new RPC module to fetch sidechain blocks. I don't have any tests yet for this RPC part - but we should definitely add some.